### ISSUE TITLE
[Bug]: xbox one s controller not work

### ISSUE THREAD

**Issue by yaya56** [PlayCover/PlayCover#708]
### Describe the bug

macbook pro m1. 16G   os 13.0.1  , playcover version 2.0.2 
connect xbox one s controller with bluetooth, Genshin game can choose the controller, then nothing happened, no button works, other origin game in steam works well . maybe same problem like #315.
try reconnect after start playcover,before playcover, before genshin start, all the same ,not work for me.

### Steps to reproduce

just not work with xbox ones controller

### Expected behaviour

xbox ones controller can work

### What version of PlayCover are you using?

2.0.2

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by zanderp25** [comment]
Did you close Steam while playing Genshin Impact? I think that it might be Steam using the controller for "Desktop Mode" or something. 

If not, does it show correctly in System Settings -> Game Controllers?

**Comment by yaya56** [comment]
> 

no. after it not work with Genshin Impact , i thought it might be some wrong with my controller,so i tried steam .

**Comment by yaya56** [comment]
> Did you close Steam while playing Genshin Impact? I think that it might be Steam using the controller for "Desktop Mode" or something.
> 
> If not, does it show correctly in System Settings -> Game Controllers?

in playcover system settings ,no xbox controller, just keyboard and mac control pad, but in game,can choose game controller, but after switch to game controller ,no button works

**Comment by zanderp25** [comment]
> playcover system settings

No, this: 

<img width="827" alt="Screenshot 2023-01-16 at 21 39 17" src="https://user-images.githubusercontent.com/13000207/212797803-e8830524-1497-4342-a146-b7f4f6005326.png">

> just keyboard and mac control pad

I think you have to turn the Game Control off for that to work

<img width="867" alt="Screenshot 2023-01-16 at 21 40 18" src="https://user-images.githubusercontent.com/13000207/212797928-8cc20d4e-c424-46ce-a5e2-ec3cbc7863e8.png">

**Comment by yaya56** [comment]
> > playcover system settings
> 
> No, this:
> 
> <img alt="Screenshot 2023-01-16 at 21 39 17" width="827" src="https://user-images.githubusercontent.com/13000207/212797803-e8830524-1497-4342-a146-b7f4f6005326.png">
> 
> > just keyboard and mac control pad
> 
> I think you have to turn the Game Control off for that to work
> 
> <img alt="Screenshot 2023-01-16 at 21 40 18" width="867" src="https://user-images.githubusercontent.com/13000207/212797928-8cc20d4e-c424-46ce-a5e2-ec3cbc7863e8.png">

i'll try it later, if still not work, i will take some picture back here , thanks

**Comment by yaya56** [comment]
<img width="715" alt="WeChat29bb4d6c9f3d2244f0623d20e06787f1" src="https://user-images.githubusercontent.com/11845986/213082829-ff788354-bb47-4836-8397-30b994d6fe87.png">
<img width="1108" alt="WeChat06418a8c6d57396b4b742d10dcff04d3" src="https://user-images.githubusercontent.com/11845986/213082859-8b3cf05e-6d33-441c-b55f-fd59953e9fe7.png">
<img width="755" alt="WeChate22c1dca9f764486963ddf52a5f58705" src="https://user-images.githubusercontent.c
<img width="755" alt="WeChat2b133c1b2a3bcd6f4cde32a03d2a6e50" src="https://user-images.githubusercontent.com/11845986/213082884-f00d98d7-4dc8-498e-b21c-fdf5df96f629.png">
om/11845986/213082873-26d70299-5399-4480-bd37-0eb1091c490c.png">

**Comment by yaya56** [comment]
<img width="715" alt="WeChat29bb4d6c9f3d2244f0623d20e06787f1" src="https://user-images.githubusercontent.com/11845986/213082959-1022a391-6f43-4a6f-b47f-edd4e72ae77d.png">
<img width="755" alt="WeChate22c1dca9f764486963ddf52a5f58705" src="https://user-images.githubusercontent.com/11845986/213082975-39d015d9-7dbb-4629-b499-07a04a9e4eb2.png">
<img width="755" alt="WeChat2b133c1b2a3bcd6f4cde32a03d2a6e50" src="https://user-images.githubusercontent.com/11845986/213082985-47286484-2e38-4eaf-b2ca-c9ac015ef962.png">
<img width="1108" alt="WeChat06418a8c6d57396b4b742d10dcff04d3" src="https://user-images.githubusercontent.com/11845986/213082997-876359f8-db9e-4d06-81e8-41951a19c444.png">
controller version is 1708,  in game can find it ,but can't move or anything

**Comment by kajarah** [comment]
@zanderp25 This is still an issue in the latest nightly, with no Steam installed.

- Xbox controller successfully paired with macOS via Bluetooth
- Controller recognized in Game Controllers in System Settings
- Game Control touch emulation is disabled
- The controller shows up in the in-game input selection menu
- No inputs (buttons, analog sticks, etc.) have any effect whatsoever

This is not an issue with the game as the game recognizes the controller.  PlayCover is not passing input through to the game.

![Game Controllers](https://cdn.discordapp.com/attachments/438465966721073173/1068721691099943053/Screenshot_2023-01-27_at_6.36.28_PM.png)
![Touch emulation](https://cdn.discordapp.com/attachments/438465966721073173/1068723153708253244/Screenshot_2023-01-27_at_6.43.58_PM.png)
![Game control settings](https://cdn.discordapp.com/attachments/438465966721073173/1068723153435643905/Screenshot_2023-01-27_at_6.43.55_PM.png)
![Game settings](https://cdn.discordapp.com/attachments/438465966721073173/1068721975985451078/Screenshot_2023-01-27_at_6.39.23_PM.png)
![PlayChain](https://cdn.discordapp.com/attachments/438465966721073173/1068722059154292796/Screenshot_2023-01-27_at_6.39.43_PM.png)

**Comment by zanderp25** [comment]
Huh, so it must be something related to the Xbox controller... Unfortunately my only Xbox controller is broken so I can't test this. I do have an X-Input controller and it works perfectly fine.

**Comment by kajarah** [comment]
@zanderp25 Can you name the specific model of controller you use?  XInput on Mac?  I thought that was a DirectX thing?  I’m willing to offer a bounty (controller hardware cost) for the complete and proper fixing of this bug.

**Comment by zanderp25** [comment]
I have 8bitdo Pro 2 and it works in Apple/PlayStation and Xinput modes. It's a really handy controller because I can use it with my Switch as well.
Xinput on a Mac behaves like an Xbox controller.

**Comment by kajarah** [comment]
Got it, thanks; I’ll check out that controller too.  Might be good to start a list of officially supported controllers.  I don’t know what exactly changes in how XInput handles the various Xbox controller models, but if it helps, I have the Bluetooth Xbox controller with a micro USB port (so it’s on the older side).

Once my work projects taper off a bit I may dive into the source code and try to help you investigate (c. early March), time permitting.

**Comment by rp4k** [comment]
i am having the same issues on my m2 mac   in my game controllers in my settings my "gamepadX" shows up fine and it works with normal Apple Arcade games on mac like nba2k23 , but when i try to use it on playcover it doesn't register the device at all

**Comment by notaLonelyDay** [comment]
have the same problem with dualsense 

<img width="680" alt="image" src="https://github.com/PlayCover/PlayCover/assets/33029221/801fedd9-5671-4b38-8d7c-03838ba422be">
<img width="555" alt="image" src="https://github.com/PlayCover/PlayCover/assets/33029221/b89f7f6b-2f06-41f7-a9c8-e5d807e5a855">
<img width="433" alt="image" src="https://github.com/PlayCover/PlayCover/assets/33029221/9db9231b-0f90-4a92-a75c-9dbc609219d4">

The game does not see controller at all

**Comment by zaithr** [comment]
this is how i did it 

 with sequia 15 even third party controller has acsess . its probably play cover is not passing through the input .try this, Update Genshin Impact and PlayCover first, then connect your controller and check for any system updates. If that doesn’t help, install Xcode, update macOS again, then launch Genshin, connect your controller, and uninstall Xcode while the game is running. This worked for me—hope it helps!"
